app/views/docs/permissions.phtml

```diff
@@ -156,8 +162,12 @@ const databases = new Databases(client, '[DATABASE_ID]');
 let promise = databases.createDocument(
     '[COLLECTION_ID]',
     {'actorName': 'Chris Evans', 'height': 183},
-    ['team:5c1f88b87435e'], // The user must be a team member to grant this permission
-    ['team:5c1f88b87435e/owner']); // The user must be a team owner to grant this permission
+    [
+        Permission.read(Role.team("5c1f88b87435e"))
+        Permission.edit(Role.team("5c1f88b87435e", "owner")),
+        Permission.delete(Role.team("5c1f88b87435e", "owner"),
```

Comment:
Can we add comment in this example too? For each like what it allows.

Also, missing comma.